### DISCUSSION TITLE
Collect academic student emails.

### DISCUSSION THREAD

**Discussion by drosofff** [ARTbio/AnalyseGenome#16] [General]
Please, add your ACADEMIC email adress bellow in this discussion, in the form of myid [at] my [dot] domain [dot] fr

I insist on the fact that Google is expecting academic email adresses. The have become touchy on this since last year.

Thanks in advance !

**Comment by drosofff** [comment]
christophe [dot] antoniewski [at] sorbonne-universite [dot] fr

**Reply by slecrom** [reply]
@drosofff how does it work for dash - and underscore _ ? Would I code my email address this way "stephane [dot] le [underscore] crom [at] sorbonne-universite [dot] fr"  instead of "stephane [dot] le_crom [at] sorbonne [dash] universite [dot] fr"?

**Reply by drosofff** [reply]
Any solution, don't worry, regex works miracles!

**Reply by Salimaiche** [reply]
Salim.Aiche@etu.sorbonne-universite.fr

**Comment by yeloett** [comment]
yelene [dot] etter [at] etu [dot] sorbonne-universite [dot] fr

**Comment by lin-ch1** [comment]
Christine [dot] lin [dot] 1 [at] etu [dot] sorbonne-universite [dot] fr

**Comment by SabineVautier** [comment]
sabine [dot] vautier [at] univ-rouen [dot] fr

**Comment by JulieBirgel** [comment]
julie [dot] birgel [at] etu [dot ]sorbonne-universite [dot] fr

**Comment by AlixMartin1** [comment]
alix [dot] martin [at] etu [dot] u-paris [dot] fr

**Comment by CoralineBernachot** [comment]
coraline [dot] bernachot [at] etu [dot] unicaen [dot] fr

**Comment by laureasselin** [comment]
lasselin [at] edu [dot] bio [dot] ens [dot] psl [dot] eu

**Comment by AstridCanal** [comment]
astrid [dot] canal [at] etu [dot] u-paris [dot] fr

**Comment by camillejarry** [comment]
camille [dot] jarry [at] u-psud [dot] fr

**Comment by zoeguilbert** [comment]
zoe.guilbert@etu.u-paris.fr

**Reply by zoeguilbert** [reply]
zoe [dot] guilbert [dot] 20 [at] etu [dot] u-paris [dot] fr

**Comment by claudiamart1s** [comment]
claudia [dot] martins [at] etu [dot] sorbonne-universite [dot] fr

**Comment by LaurineBourel** [comment]
laurine [dot] bourel77 [at] etu [dot] u-paris [dot] fr

**Comment by MarieMassier** [comment]
marie [dot] massier [at] etu [dot] sorbonne-universite [dot] fr

**Comment by LeilaNeu** [comment]
leila[dot]qebibo[at]etu[dot]u-paris[dot]fr

**Comment by Salimaiche** [comment]
Mon adresse mail pro : Salim.Aiche@etu.sorbonne-universite.fr

**Comment by slecrom** [comment]
@drosofff c'est bon les 15 participant•e•s du cours analyse des génomes avons rempli nos adresses de courrier électronique professionnelle dans la discussion.
Bonne soirée.

**Comment by drosofff** [comment]
Thank you everyone, the application to Google Inc was submitted yesterday ! 🎉
